pgd_splicer/ProcessPDBTask.py

```diff
@@ -403,6 +403,58 @@ def atom_noamino(s):
     return s.startswith("ATOM ") and not amino_present(s)
 
 
+class OccupancySelect(Select):
+    """
+    This class implements the occupancy awareness selection.
+
+    Each disordered residue contains one or more disordered atom structures.  These structures contain one or more atoms, each with its own altloc and occupancy values.  As each atom associated with a particular altloc has its own occupancy value, the altloc with the highest average occupancy value is identified.  Atoms with other altloc values are removed from the structure.
```

Comment:
This file has 173 PEP8 violations, most of which are more interesting than line-length.

I will wrap the comments. :-)